modules/serverless/pages/service-mesh/common-service-mesh-network-isolation.adoc

```diff
@@ -0,0 +1,532 @@
+= Use {smproductname} to isolate network traffic with {serverlessproductname}
+:compat-mode!:
+// Metadata:
+:description: Use {smproductname} to isolate network traffic with {serverlessproductname}
+
+{smproductshortname} can be used to isolate network-traffic between tenants on a shared OpenShift cluster using {smproductshortname} `AuthorizationPolicy` resources.
+{serverlessproductname} can also leverage this, using several {smproductshortname} resources.
+
+.Tenants
+A tenant is a group of one or multiple OpenShift projects that can access each other over the network on a shared OpenShift cluster.
+
+.High-level architecture
+The high-level architecture consists of `AuthorizationPolicies` in the `knative-serving`, `knative-eventing` and the tenants namespaces, while all components are part of the {smproductshortname}. The injected {smproductshortname} sidecars take care of enforcing those rules to isolate network traffic between tenants.
+
+*Knative Serving*
+
+image::service-mesh/multi-tenancy-service-mesh-Serving.drawio.svg[]
+
+. Workloads in project tenant-2 can not call workloads in project tenant-1 directly. Requests are rejected in the Istio proxies of tenant-1 workloads.
+. Workloads in project tenant-2 can not call workloads in project tenant-1 via ingress-gateway. Requests are rejected in the Istio proxies of tenant-1 workloads.
+. Workloads in project tenant-2 can not call workloads in project tenant-1 via activator. Requests are rejected in the Istio proxy of the activator component. Note: as ingress-gateway is allowed to call activator, this includes requests from project tenant-2 via ingress-gateway to activator.
+. 🔒 `AuthorizationPolicies` are applied in this project
```

Comment:
aren't also applied in `tenant-2` project?

Reply:
Yes they are also applied there, but the diagram only shows examples from tenant-1 to tenant-2. Not sure if adding helps or not.

Reply:
I added it, then we are consistent with the Eventing diagrams.